neurom/geom/tests/test_transform.py

```diff
@@ -28,14 +28,30 @@
 
 import neurom.geom.transform as gtr
 from neurom.core.dataformat import COLS
-from neurom.core.tree import val_iter, ipreorder
+from neurom.core.tree import val_iter, ipreorder, Tree, make_copy
+from neurom.core.neuron import Neuron, make_soma
 from neurom.ezy import load_neuron
 from nose import tools as nt
 from itertools import izip
 import numpy as np
+from copy import copy
 
+TREE = Tree(np.array([0.0, 0.0, 0.0, 1.0, 1, 1, 2]))
```

Comment:
I have a question. Is there an advantage behind having multiple type support for the neurites and neurons? I mean why someone would need to use tuples instead of numpy arrays, for example, for the neurite values? 

I can see why the generalised tree structure can store a variety of types, but the neurites are a specific version of the tree and the transformations are intended for the latter which also need to start with x, y, z. Thus, not every tree can be transformed.